src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRPollingLogAction.java

```diff
@@ -16,13 +16,13 @@
  * @author Alina Karpovich
  */
 public class GitHubPRPollingLogAction implements Action {
-    private transient AbstractProject<?, ?> project;
+    private transient Job<?, ?> project;
 
-    public GitHubPRPollingLogAction(AbstractProject<?, ?> project) {
+    public GitHubPRPollingLogAction(Job<?, ?> project) {
         this.project = project;
     }
 
-    public AbstractProject<?,?> getOwner() {
+    public Job<?,?> getOwner() {
```

Comment:
breaks binary compatibility. You need to bump the major version or to introduce bridge methods